hue_lo: 99
hue_hi: 119
sat_lo: 200
sat_hi: 300
val_lo: 48
val_hi: 228
timing_threshold: 5.0
